### ISSUE TITLE
heading-style: settext headings vs front matter

### ISSUE THREAD

**Issue by Elchi3** [remarkjs/remark-lint#211]
I've set `["remark-lint-heading-style", "atx" ]` so that I can consistently replace any settext style headings with atx style headings. However, the linter also complains that my front matters are settext style headings. Is this a known issue?

```md
---
foo: bar
---

# `baz`

```

2:1-3:4  warning  Headings should use atx                                 heading-style             remark-lint

**Comment by Elchi3** [comment]
Never mind, just saw https://github.com/remarkjs/remark-frontmatter and installing it seems to fix this. Loving remark so far btw! Thanks for all your work :+1:

**Comment by wooorm** [comment]
Thanks, and thank *you* for using and enjoying the ecosystem! ✨

Yep, frontmatter syntax isn’t turned on by default. remark-frontmatter does that. Although most of the other plugins should handle the yaml nodes!
